src/Form/Form.js

```diff
@@ -580,10 +581,30 @@ class Form {
 
     shouldOpenTooltip (e, input) {
         if (!isPotentiallyViewable(input)) return false
-        if (this.device.globalConfig.isApp) return true
-        if (this.device.globalConfig.isWindows) return true
+
+        // Always open if the user has clicked on the Dax icon
         if (isEventWithinDax(e, input)) return true
-        if (this.device.inContextSignup?.isAvailable()) return false
+        if (this.device.globalConfig.isWindows) return true
```

Comment:
Not a blocker, but ideally we'd have an `isDesktopApp` in the globalConfig and here we'd check for: `isDesktopApp && !isIncontextSignupAvailable` or something like that, rather than relying on this being a specific platform.

Reply:
@GioSensation This is a weird diff, but that line wasn't added in this PR, it's from a few lines up (but +1 to removing platform specific checks)
This PR moved the `isApp` check down and essentially facilitates the check of not opening the tooltip if, in the macOS app, we don't have anything to autofill except in-context signup.